src/openrct2/scenario/Scenario.h

```diff
@@ -282,8 +283,8 @@ struct rct_s6_data
     uint8_t next_weather;
     uint8_t temperature;
     uint8_t next_temperature;
-    uint8_t current_weather_effect;
-    uint8_t next_weather_effect;
+    WeatherEffectType current_weather_effect;
```

Comment:
Could I be annoying and request that these stay as `uint8_t` this struct should really be moved into the RCT2 header as its a RCT2 specific struct. We should always be validating the values before taking a value from the save and putting it into memory by keeping the types different this will ensure they do not directly convert.

Reply:
Oh, I thought I already changed it back, sorry, I missed this. I will correct it of course!